docs/BRP-LO GBA mapping - Personen gba.csv

```diff
@@ -0,0 +1,144 @@
+veldnaam;;;;;categorie;element;Leveringsvorm;Buitengemeentelijke autorisatie;Beperkt zoekresultaat;
```

Comment:
volgens mij hadden we afgesproken dat we de mapping documenten zouden verwijderen, om dubbel onderhoud en inconsistenties te voorkomen. Is nu het moment om dat te doen?